packages/yew-test-runner/src/procedural.rs

```diff
@@ -0,0 +1,306 @@
+//! A procedural style setup for a test runner
+
+use std::fmt;
+
+use web_sys::Element;
+use yew::{AppHandle, Html, Renderer};
+
+use crate::scaffold::{TestScaffold, TestScaffoldProps};
+
+/// The test runner controls a piece of the DOM on which your components are mounted.
+///
+/// Use the [`TestCase`] trait to access the testing functionality.
+pub struct TestRunner {
+    test_app: Option<AppHandle<TestScaffold>>,
+    parent: Element,
+}
+
+impl fmt::Debug for TestRunner {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("TestRunner").finish_non_exhaustive()
+    }
+}
+
+impl Drop for TestRunner {
+    fn drop(&mut self) {
+        if let Some(app) = self.test_app.take() {
+            app.destroy();
+        }
+    }
+}
+
+/// Helper trait for things that can be more-or-less treated as a [`TestRunner`].
+///
+/// Automatically gets an implementation of [`TestCase`].
```

Comment:
```suggestion
/// Automatically gets an implementation of [`TestRunner`].
```

I think you meant `TestRunner`, not `TestCase`